zipline/data/us_equity_pricing.py

```diff
@@ -978,12 +978,21 @@ def calc_dividend_ratios(self, dividends):
         equity_daily_bar_reader = self._equity_daily_bar_reader
 
         effective_dates = full(len(amounts), -1, dtype=int64)
+
         calendar = self._calendar
+
+        # Calculate locs against a tz-naive cal, as the ex_dates are tz-
+        # naive.
+        tz_naive_calendar = calendar.tz_localize(None)
```

Comment:
Attempted this at first in https://github.com/quantopian/zipline/pull/1510/commits/189b868f7f66b5b906f162121ef621f0b224c51b, but `get_indexer` appears to drop the tz info, which throws off the comparison, so going with this workaround for now.